### ISSUE TITLE
Take default Python version from NEST installation

### ISSUE THREAD

**Issue by kappeld** [IGITUGraz/spore-nest-module#25]
I would suggest to change the default behavior of the installation script to take the Python version from the nest installation. `nest-config` recently introduced the feature to access the python version and path, so it should be quite easy to adapt the cmake script. Since in all practical cases the python version of SPORE and NEST should be the same this is more convenient for most users. We can still allow to overwrite the default using `-Dwith-python'.

see:

```
nest-config  --python-executable # print full path to Python interpreter used
nest-config  --python-version     # print Python version string for interpreter
```

**Comment by mhoff** [comment]
Agreed. For the majority of users, if not for all of them, using the python version of NEST as default should be correct.

**Comment by kappeld** [comment]
Ok, great. I quickly tested this and implemented the following behavior:

- The NEST version is now detected by cmake.
- Python version is parsed from nest-config and overwrites the default.
- if -Dwith-python is set it overwrites the default nest version.
- if NEST is not found the installation aborts (we cannot even compile SPORE without a nest installation).
- If NEST version is below minimum a warning is displayed but installation resumes.

I think this change makes the installation smoother but does not reduce the flexibility. We also have to adapt the REAMDE and wiki pages. I will make a PR later.
